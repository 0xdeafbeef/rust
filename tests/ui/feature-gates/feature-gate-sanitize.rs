#[sanitize(address = "on")]
//~^ ERROR the `#[sanitize]` attribute is an experimental feature
fn main() {
}
