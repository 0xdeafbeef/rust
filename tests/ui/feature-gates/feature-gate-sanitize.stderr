error[E0658]: the `#[sanitize]` attribute is an experimental feature
  --> $DIR/feature-gate-sanitize.rs:1:1
   |
LL | #[sanitize(address = "on")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #39699 <https://github.com/rust-lang/rust/issues/39699> for more information
   = help: add `#![feature(sanitize)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0658`.
